3scale/porta
# Unwanted text in product settings RadioGroup option

**Which issue(s) this PR fixes** 

[THREESCALE-8556 Unwanted text in product settings RadioGroup option](https://issues.redhat.com/browse/THREESCALE-8556)

**Verification steps** 

1. Navigate to Product setting page (/apiconfig/services/{product_id}/settings).
2. Confirm Istio option has only "Istio". 

![Captura de Pantalla 2022-07-29 a las 17 43 13](https://user-images.githubusercontent.com/72191439/181795584-3e1fd9c2-b93f-4e68-8106-18dbc96a0de0.png)


**Special notes for your reviewer**:

After some considerations, I've decided to delete the unwanted text form the `en.yml` file instead of hiding it. That text was hidden at version 2.7 and has been like that since, so it made more sense to simply delete it.
